substrate/frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -245,6 +245,33 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 			}
 		});
 
+	let deprecation_status = if let Some(call) = def.call.as_ref() {
+		crate::deprecation::get_deprecation(&quote::quote! {#frame_support}, &call.attrs)
+			.expect("Correctly parse deprecation attributes")
```

Comment:
I don't think syn will actually enforce the correct parsing of `deprecated`. It would be better not to panic.

Reply:
Yep, that makes sense to me! We can turn these into `compile_err`

Reply:
done